rasa_core/processor.py

```diff
@@ -287,10 +287,24 @@ def _run_action(self, action, tracker, dispatcher):
             logger.error(e, exc_info=True)
             events = []
         self._log_action_on_tracker(tracker, action.name(), events)
+        self._log_bot_utterances_on_tracker(tracker, dispatcher)
         self._schedule_reminders(events, dispatcher)
 
         return self._should_predict_another_action(action.name(), events)
 
+    def _log_bot_utterances_on_tracker(self, tracker, dispatcher):
+        # type: (DialogueStateTracker, Dispatcher) -> None
+
+        if dispatcher.bot_message is not None:
```

Comment:
also needs to reset the `dispatcher.bot_message` otherwise we will log the message again for the next action.